Source/WebKit/UIProcess/WebPageProxy.h

```diff
@@ -857,6 +857,8 @@ class WebPageProxy final : public API::ObjectImpl<API::Object::Type::Page>
     void scrollingNodeScrollWillStartScroll(WebCore::ScrollingNodeID);
     void scrollingNodeScrollDidEndScroll(WebCore::ScrollingNodeID);
 
+    WebCore::FloatSize overrideScreenSize();
```

Comment:
Should be `const`

Reply:
Also "override" a verb. This should be `overriddenScreenSize()`

Reply:
Sorry I missed these comments :/ But I can file a follow-up!

The method wasn't introduced here but are you suggesting we rename/const it all the way to PlatformScreen?